### PR TITLE
add some settings to make docker devstack build

### PR DESCRIPTION
**Pull request by thraxil**
This fixes a couple errors encountered while building the docker image for devstack

I set a default for `private_requirements_file` to the same as we set in our usual base settings.

Is there a better way to make sure that the `EDXAPP_EDXAPP_SECRET_KEY` variable is defined but make sure that it definitely gets overridden in production setups? (for the purposes of getting devstack to build, I'm not worried about it).

### CODE REVIEW

**Comment by OmarIthawi** [comment]
@thraxil Ax currently generates it for new deployments so I wouldn't worry about that.

File: docker/build/edxapp/ansible_overrides.yml [hunk 1]
```diff
@@ -40,3 +40,4 @@ EDXAPP_XQUEUE_URL: 'http://edx.devstack.xqueue:18040'
 
 EDXAPP_ENABLE_EDXNOTES: true
 EDXAPP_EDXNOTES_INTERNAL_API: 'http://edx.devstack.edx_notes_api:18120/api/v1'
+EDXAPP_EDXAPP_SECRET_KEY: "change me"
```
Comment (OmarIthawi) [inline docker/build/edxapp/ansible_overrides.yml]: Thanks for fixing this. Do you mind cherry-picking my upstream pull request? So we can avoid conflicts in the future:

 - https://github.com/edx/configuration/pull/4778

**Review by OmarIthawi — APPROVED** [review]
LGTM! Thanks @thraxil

File: docker/build/edxapp/ansible_overrides.yml (hunk 1 above)
Comment (thraxil) [reply docker/build/edxapp/ansible_overrides.yml]: @OmarIthawi done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+
+- Role: analytics_api
+  - Added `ANALYTICS_API_CORS_ORIGIN_WHITELIST` to allow CORS whitelisting of origins.
+
+- Role: edxapp
+  - `EDXAPP_X_FRAME_OPTIONS` added in studio to prevent clickjacking.
+
+- Role: edxapp
+  - Added `EDXAPP_X_FRAME_OPTIONS` to prevent click jacking in LMS.
+>>>>>>> bfa46d5... Remove the default edxapp secret key to ensure secure defaults
```
Comment (OmarIthawi) [inline CHANGELOG.md]: Seems like a merge conflict to me.

**Review by OmarIthawi — CHANGES_REQUESTED** [review]
Thanks @thraxil! The PR looks good now. Please fix the merge conflicts in `CHANGELOG.md`

File: CHANGELOG.md (hunk 2 above)
Comment (thraxil) [reply CHANGELOG.md]: Argh. Yep, missed one.

Comment (thraxil) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
 - Use Ansible 2.3.1.0 so that we can do shallow clones of tags.
 
+- Role: edx_django_service
```
Comment (OmarIthawi) [inline CHANGELOG.md]: There's only one entry in the `CHANGELOG.md` from the original pull request: 

https://github.com/edx/configuration/pull/4778/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1e

Could you please keep:

```
- Role: edxapp
  - `EDXAPP_EDXAPP_SECRET_KEY` no longer has a default value
```

While removing the others?

**Review by OmarIthawi — CHANGES_REQUESTED** [review]
Thanks @thraxil!  One more change regarding the conflict.

File: CHANGELOG.md (hunk 3 above)
Comment (thraxil) [reply CHANGELOG.md]: fixed

**Review by OmarIthawi — APPROVED** [review]
@thraxil this is great! Thank you.

Let's get this merged!

**Pull request merged**